app/templates/_browserslistrc

```diff
@@ -1,6 +1,4 @@
-IE >= 10
-Firefox >= 54
-Chrome >= 60
-Safari >= 10
-iOS >= 10.2
-Android >= 4.4.4
+IE >= 11
+Firefox >= 64
+Chrome >= 70
+Safari >= 12
```

Comment:
We're missing Edge, iOS and ChromeAndroid. Also I'm also thinking maybe we should use things like `last N versions` or `>N% in US` to escape maintenance burden? Just an idea 😄

Reply:
I like the idea and it does eliminate some maintainability. However, there are two reasons why it's been kept fixed. First its usually fixed in our contracts towards customers as we don't know what potentially changes can come in newer versions. So to avoid any unnecessary risks historically we've kept version as such. Second is that in order to update this the person responsible is forced to look into the contract/agreement document or at least dig it up 🙊

Reply:
Alright, so these versions are actually fixed in a contract and that's why we have them here (hopefully my understood is correct). Then, yea, it makes sense 😄

My understanding was that projects might have different requirements, therefore we can use `last N versions` as a default and then person kicking off the project has to update the config accordingly.

BTW what about mobile browsers? 📱

Reply:
Well, these versions are sort of fixed in a contract.. we tend to update them as we go or perhaps if whenever we are lucky and have a dialog with the customer beforehand. I would really like the `last N version` So perhaps it's better to let the person responsible be more specific and be more generic out of the box. 

So, the question is how would it look if we did so.. as I see it we don't' want last lets say 2 versions of every browser. and as for mobile? Well, aren't they just the same as desktops now days?

Reply:
> and as for mobile? Well, aren't they just the same as desktops now days?

As I see from Can I use stats, there might be a mismatch between desktop and mobile browser in terms of features. For example custom properties were available in Safari 9.1 on desktop, but not on mobile (on mobile only in Safari 9.3). Same thing with Chrome. So, I thought, we might include them just to be 100% sure 🙂 Also Browserslist does [differentiate them](https://github.com/browserslist/browserslist#browsers).

> So, the question is how would it look if we did so..

That's sort of what I use by default:

```
last 5 Chrome versions
last 5 ChromeAndroid versions
last 2 Edge versions
Explorer 11
last 5 Firefox versions
Firefox ESR
last 2 iOS major versions
last 2 Safari major versions
```

[Result on browserl.ist](https://browserl.ist/?q=last+5+Chrome+versions%2C+last+5+ChromeAndroid+versions%2C+last+2+Edge+versions%2C+Explorer+11%2C+last+5+Firefox+versions%2C+Firefox+ESR%2C+last+2+iOS+major+versions%2C+last+2+Safari+major+versions)

Reply:
@racse1 Long time, let's wrap this one :)  I've adjusted it a little but it still has about 80% global coverage.

```
Explorer 11
last 2 Edge versions
last 4 Chrome versions
last 4 ChromeAndroid versions
last 4 Firefox versions
last 2 iOS major versions
last 2 Safari major versions
```